plugins/woocommerce-admin/client/core-profiler/use-loader-stages.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import LightBulb from './assets/images/loader-lightbulb.svg';
+import { Stages } from './pages/Loader';
+
+export const useStages = ( key: string ): Stages => {
+	switch ( key ) {
+		case 'default':
+		default:
+			return [
+				{
+					title: __( 'Turning on the lights', 'woocommerce' ),
+					progress: 80,
```

Comment:
This set of settings make the "loading screen" look static, I thiiiiink the intent is for the progress bar to move from 0-100 within the loading time?

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/38019/commits/58d3750624b13f1104c4a3d4aa2822d94fa87f77, but I'm interested if there is a cleaner way 🤔